sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/config/AdminConfig.java

```diff
@@ -11,6 +13,8 @@
 
   private Security adminSecurity = new Security();
 
+  private Map<String, String> config = new HashMap<>();
```

Comment:
Does the general `KafkaConfiguration` need a flexible `config` map as well?

Reply:
Yes. This could also obsolete this addition in the `AdminConfig`. We will need for the SSL stuff anyway (#589) so it might be a good idea to already add it here. This also simplifies the part with the custom SASL mechanism in the Readme.